roles/foreman_provisioning/tasks/main.yml

```diff
@@ -9,12 +14,6 @@
   set_fact:
     foreman_provisioning_smart_proxy: "{{ foreman_provisioning_smart_proxy_json.stdout|from_json }}"
 
-- name: 'install libvirt gem'
```

Comment:
You need to keep it. Without ruby-libvirt gem, proxy cannot have dns and dhcp feature for libvirt.

Reply:
```foreman-libvirt``` will pull this in so it's redundant.

Reply:
Correction, it is needed since foreman-libvirt pulls in the SCL version where this is the non-SCL version for the proxy.